### PR TITLE
Synchronize key-value pairs across workers

### PR DESCRIPTION
**Pull request by dharmit**

### CODE REVIEW

**Comment by bamachrn** [comment]
#dotests

**Comment by bamachrn** [comment]
#dotests-debug

**Comment by bamachrn** [comment]
#dotests-debug

**Comment by bamachrn** [comment]
#dotests-debug

**Comment by bamachrn** [comment]
#dotests-debug

File: container_pipeline/trigger_dockerfile_lint.py [hunk 1]
```diff
@@ -81,20 +48,20 @@ def trigger_dockerfile_linter(appid, jobid, repo_url, repo_branch,
         print "==> Sending Dockerfile linter failure email"
         response = {
```
Comment (bamachrn) [inline container_pipeline/trigger_dockerfile_lint.py]: We should remove this job creation and use the job variable changing the action and setting msg. May be we can add `msg` to dictionary also. We can take care of these while re factoring the mail service as well. (All the separate job creation left in this PR, are related to `notify_user`)

Comment (dharmit) [reply container_pipeline/trigger_dockerfile_lint.py]: As discussed, let's keep this in a separate PR wherein we refactor the `notify_user` piece.

**Pull request merged**